Dockerfile

```diff
@@ -12,14 +12,14 @@ COPY --from=drupal /usr/local/etc/php/conf.d/* /usr/local/etc/php/conf.d/
 
 COPY --from=composer/composer:2-bin /composer /usr/local/bin/
 
-# https://www.drupal.org/project/drupal/releases/10.1.2
-ENV DRUPAL_VERSION 10.1.2
+# https://www.drupal.org/project/drupal/releases
+ENV DRUPAL_VERSION 10.1
 
 WORKDIR /opt/drupal
 # Copy from the official image when Drupal 10.0 will be released
```

Comment:
Cannot we do this instead now?

Reply:
The official images are here: https://hub.docker.com/_/drupal

Something like `COPY --from=drupal ---link /opt/drupal /opt/drupal` should do the trick and be always up to date.

(https://github.com/docker-library/drupal/blob/master/Dockerfile.template)

Reply:
I'll check this